src/app/services/settings.service.ts

```diff
@@ -57,10 +55,6 @@ export class SettingsService {
     });
   }
 
-  isExperimentalEnabled(): boolean {
```

Comment:
@ziajka  Did you mean not removing options in the settings service?

Reply:
We may want to keep it